common-template/src/main/resources/templates/email/Patch/beta/dailyEmailBody.html

```diff
@@ -0,0 +1,49 @@
+{@boolean renderSection1=true}
+{@boolean renderTitleRightPart=true}
+{#include email/Common/insightsEmailBodyLightV2}
+{#content-title-section1}
+    Advisories
+{/content-title-section1}
+{#content-title-counter}
+({action.context.total_advisories})
+{/content-title-counter}
+{#content-title-right-part}
+{action.context.total_advisories}
+{/content-title-right-part}
+{#content-body-section1}
+<p style="{body-section-p-style}"> There are {action.context.total_advisories} new advisories affecting your systems:</p>
+    {#for key in action.context.patch.keySet()}
+        {#if action.context.patch.get(key).size() > 0}
+            {#if !key_isFirst}
+                <div style="height: 24px">&#xA0;</div>
+            {/if}
+            <table style="border: solid 1px #b8bbbe; border-radius: 16px; padding: 0 8px 0 8px; width: 100%;"><tr><td>
+            <table style="{body-section-table-style}">
```

Comment:
Is there no other option than a double `table` here?

Reply:
I'm not adding this comment on all occurrences of this approach, but my question applies to all of them.